packages/strapi-admin/admin/src/components/EventInput/tests/EventRow.test.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import EventRow from '../EventRow';
+
+describe('<EventRow />', () => {
```

Comment:
It would be great if you could add some test for the `handleChangeAll` and `handleChange` you could `@testing-library` so it is easier